lib/algmat.gi

```diff
@@ -1490,6 +1490,8 @@ InstallGlobalFunction( EmptyMatrix, function( char )
     SetAdditiveInverse( mat, mat );
     SetDimensionsMat( mat, [ 0, 0 ] );
     SetLength( mat, 0 );
+    SetNumberRows( mat, 0 );
+    SetNumberColumns( mat, 0 );
```

Comment:
Thanks. BTW, one could improve this code even more by replacing `Objectify` with `ObjectifyWithAttributes`, which would avoid immediate method overhead for setting all these attributes. Alas, that can be done in a later PR.

Reply:
I couldn't work out how to do `Set{Additive}Inverse(mat, mat)` inside of an `ObjectifyWithAttributes`, but I did the others.

Reply:
Beautiful, thank you so much!